docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -250,9 +253,36 @@ your installation path. You can use the following settings, if you installed R a
 - Windows: `C:\Program Files\R\R-3.6.X'` (where X is the version you used to install the above libraries)
 - macOS: `/Library/Frameworks/R.framework/Versions/3.6/Resources`
 
+##### KNIME OpenMS plugin
+
 You are now ready to install the OpenMS nodes.
-- Open KNIME.
-- Click on **Help** > **Install New Software**
+- In KNIME, click on **Help** > **Install New Software**
+
+You now have to choose an _update site_ to install the OpenMS plugin from. Which _update site_ to choose depends on if you received an USB stick
+in a hands-on Tutorial or if you are doing this Tutorial online.
+
+::::{tab-set}
+
+:::{tab-item} Online
+:sync: online
+
+To install the OpenMS KNIME plugin from the internet, do the following:
+
+1. From the '**Work with**:' drop-down list, select the _update site_ 'KNIME Community Extensions (Trusted) - https://update.knime.com/community-contributions/trusted/4.6'
+2. Now select the following plugin from the "KNIME Community Contributions - Bioinformatics & NGS" category
+   - OpenMS
+3. Click on **Next**  and follow the instructions and after a restart of KNIME the OpenMS nodes will be available in the Node repository under
+  "Community Nodes".
+  
+```{note}
+If this does not work for you, report it and while waiting for a reply/fix, try to use an _update site_ of an older KNIME version by editing the KNIME
+version number in the URL or by using our inofficial _update site_ at https://abibuilder.cs.uni-tuebingen.de/archive/openms/knime-plugin/updateSite/release/latest
+```
+
+:::
+
+:::{tab-item} USB
```

Comment:
seen this tab ::::{tab-set}  :::{tab-item}  and check the final outcome works fine.